### ISSUE TITLE
[seo-magic] Meta Keyword and Meta Description not auto generating

### ISSUE THREAD

**Issue by ptarmic** [getgrav/grav-premium-issues#21]
Meta Title is working fine. 
The result is the same on pages and modulars

![image](https://user-images.githubusercontent.com/72616419/105523728-a8390d00-5cd6-11eb-809e-19fd90494c86.png)

I don't see any errors when reprocessing seo data.

I had some text in the metadata section of ` /user/config/site.yaml` That text was showing in the auto-generated section of the seo field but when I deleted that text the auto generated fields are now blank. Also when I deleted the whole description field from the site.yaml I got a hard crash viewing a page.
![image](https://user-images.githubusercontent.com/72616419/105526067-d23ffe80-5cd9-11eb-8578-15888ee5bdbd.png)

I'm on Grav 1.7.3

**Comment by rhukster** [comment]
Do you have any password protection on the site? can you reach the sitemap directly in the browser?

**Comment by ptarmic** [comment]
Nope to both. No password protection at all and sitemap works fine.
The site is behind ssl offloading so the links show as http but this hasn't been an issue with seo magic before.

This was working when I first installed the plugin (on the RC version of 1.7 that was out when the plugin was released) and I've only now noticed it's not working so possibly this is related to either grav or seo updates recently?
